dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/HostUpdateProcessor.java

```diff
@@ -45,17 +45,17 @@
     /**
      * task callback service
      */
-    private final TaskCallbackService taskCallbackService;
-
-    public HostUpdateProcessor() {
-        this.taskCallbackService = SpringApplicationContext.getBean(TaskCallbackService.class);
-    }
+    private TaskCallbackService taskCallbackService;
```

Comment:
Just add `@Autowired` might be better?

Reply:
I just tested it on my Linux server.```@Autowired``` can't solve this problem. The error message in the log remains the same.

Reply:
It's strange, `@Autowired ` can work well on my computer.

Reply:
It' odd. This error just happened today.I've test this on two different linux server with both ```mvn``` and ```mvnw``` command.
1.ubuntu 20.04
2.ubuntu 18.04

Reply:
OK, anyway, the pr has merged, it should be work now.